invenio_oauthclient/utils.py

```diff
@@ -186,14 +188,21 @@ def oauth_register(form, user_info=None, precedence_mask=None):
             filter_user_info(user_info, precedence_mask or default_mask)
             patch_dictionary(form_data, user_info)
 
-        # re-populate the form after appyling the precedence mask and
+        # re-populate the form after applying the precedence mask and
         # convert the form data to user model's data
         data = fill_form(form, form_data).to_dict()
         # see https://github.com/inveniosoftware/invenio-oauthclient/issues/275
         remove_none_values(data)
         # remove the CSRF tokens to avoid unexpected keyword arguments
         remove_csrf_tokens(data)
-        user = register_user(**data)
+
+        auto_confirm = signup_options.get("auto_confirm", False)
+        # auto_confirm sets the `confirmed_at` field
+        if auto_confirm:
+            data["confirmed_at"] = datetime.now()
```

Comment:
```suggestion
            data["confirmed_at"] = datetime.utcnow()
```

Based on [Flask-Security's `DATETIME_FACTORY`](https://github.com/inveniosoftware/flask-security-fork/blob/3001b0785d9ee769dae52484657e37e2733505db/flask_security/core.py#L124).

Reply:
You are totally right, fixed it, thank you!